server/src/main/kotlin/suwayomi/tachidesk/server/ServerSetup.kt

```diff
@@ -79,7 +79,7 @@ fun applicationSetup() {
         } else {
             setLogLevel(Level.INFO)
         }
-    })
+    }, ignoreInitialValue = false)
```

Comment:
is this 
```
        // set log level early
        if (debugLogsEnabled(config)) {
            setLogLevel(Level.DEBUG)
        }
```
still needed in the ConfigManger with this change?

I believe the issue with the debug logs is because I forgot to set the default log level, which was INFO before in the logback config file, when I added the file logging

Reply:
I think it still is, configs would get initialized and there are some logs in the config manager that should be outputted